e2e/explorer/api/src/app/etherscan/account.ts

```diff
@@ -59,3 +59,20 @@ export function txList(params: GetParams<TxList>): TransactionReceipt[] {
   const offset = Math.min(params.offset, 10000);
   return sorted.slice(offset*(page-1), offset*page);
 }
+
+export function getMinedBlocks(params: GetParams<BlockMined>) {
+  const { address, blocktype, page, offset } = params;
+  if (!address) throw new Error('Error! Missing or invalid Action name');
+
+  const minedBlocks = blocks
+    .filter(block => {
+      if (block.miner !== address) return false;
+    })
```

Comment:
You don't need a `if` statement in a filter if it's only for one validation : 
```suggestion
  const minedBlocks = blocks
    .filter(block => block.miner !== address)
```